src/patternfly/components/Label/label-template-variants.hbs

```diff
@@ -42,6 +55,22 @@
     label-text--attribute='style="--pf-v5-c-label__text--MaxWidth: 38ch"'
     label-icon--value="info-circle"}}
 
+{{> label
+    label--isRemovable=true
+    label--id=(concat label-template-variants--id '-link-disabled')
+    label--IsDisabled=true
+    label-text--value=(concat label-template-variants--title ' link removable (disabled)')
+    label-content--IsLink=true
+    label-icon--value="info-circle"}}
```

Comment:
Similar to [my comment in the v6 PR ](https://github.com/patternfly/patternfly/pull/6445#discussion_r1532104508)about disabled link labels.